db/migrate/20231111205621_add_added_automatically_to_join_models.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class AddAddedAutomaticallyToJoinModels < ActiveRecord::Migration[7.1]
+  def up
+    add_column :enchantables_enchantments,
+               :added_automatically,
+               :boolean,
+               default: false
+
+    add_column :potions_alchemical_properties,
+               :added_automatically,
+               :boolean,
+               default: false
+
+    # rubocop:disable Rails/SkipsModelValidations
+    EnchantablesEnchantment.update_all(added_automatically: true)
+    PotionsAlchemicalProperty.update_all(added_automatically: true)
```

Comment:
It doesn't really matter what we set these to since there are no `PotionsAlchemicalProperty` models in the prod database right now. For `EnchantablesEnchantment` it should be set to `true` since all of these models correspond to automatically-created canonical enchantables.